docs/release-notes.rst

```diff
@@ -13,7 +13,7 @@ Release Notes
     * Add Redis 4 compatability fix to CLUSTER NODES command (See issue #217)
     * Fixed bug with command "CLUSTER GETKEYSINSLOT" that was throwing exceptions
     * Added new methods cluster_get_keys_in_slot() to client
-    * Fixed bug with `StrictRedisCluster.from_url` that was ignoring the `readonly_mode` parameter
+    * Fixed bug with `RedisCluster.from_url` that was ignoring the `readonly_mode` parameter
```

Comment:
Should I leave this one be?  I was going through docs and it made sense to change it, 'cause it no longer references the right thing.